src/sentry/migrations/0919_dashboard_starred_add_position_column_and_constraint.py

```diff
@@ -0,0 +1,57 @@
+# Generated by Django 5.2.1 on 2025-06-02 19:31
+
+import django.db.models.deletion
+from django.db import migrations, models
+
+import sentry.db.models.fields.foreignkey
+from sentry.new_migrations.migrations import CheckedMigration
+
+
+class Migration(CheckedMigration):
+    # This flag is used to mark that a migration shouldn't be automatically run in production.
+    # This should only be used for operations where it's safe to run the migration after your
+    # code has deployed. So this should not be used for most operations that alter the schema
+    # of a table.
+    # Here are some things that make sense to mark as post deployment:
+    # - Large data migrations. Typically we want these to be run manually so that they can be
+    #   monitored and not block the deploy for a long period of time while they run.
+    # - Adding indexes to large tables. Since this can take a long time, we'd generally prefer to
+    #   run this outside deployments so that we don't block them. Note that while adding an index
+    #   is a schema change, it's completely safe to run the operation after the code has deployed.
+    # Once deployed, run these manually via: https://develop.sentry.dev/database-migrations/#migration-deployment
+
+    is_post_deployment = False
+
+    dependencies = [
+        ("sentry", "0918_sentry_release_arrayfield"),
+    ]
+
+    operations = [
+        migrations.AlterUniqueTogether(
+            name="dashboardfavoriteuser",
+            unique_together=set(),
+        ),
+        migrations.AddField(
+            model_name="dashboardfavoriteuser",
+            name="organization",
+            field=sentry.db.models.fields.foreignkey.FlexibleForeignKey(
+                null=True, on_delete=django.db.models.deletion.CASCADE, to="sentry.organization"
+            ),
+        ),
+        migrations.AddField(
+            model_name="dashboardfavoriteuser",
+            name="position",
+            field=models.PositiveSmallIntegerField(null=True),
+        ),
+        migrations.SeparateDatabaseAndState(
+            state_operations=[
+                migrations.AddConstraint(
+                    model_name="dashboardfavoriteuser",
+                    constraint=models.UniqueConstraint(
+                        fields=("user_id", "dashboard"),
+                        name="sentry_dashboardfavoriteuser_unique_favorite_per_dashboard",
```

Comment:
```suggestion
                        name="sentry_dashboardfavoriteuser_user_id_dashboard_id_2c7267a5_uniq",
```

We need to retain the old name, or we won't be able to drop the unique constraint in the future unless we also drop the table.